integration-tests/eks/validateResources_test.go

```diff
@@ -76,14 +90,17 @@ func TestOperatorOnEKs(t *testing.T) {
 	//Validating the services
 	services, err := ListServices(nameSpace, clientSet)
 	assert.NoError(t, err)
-	assert.Len(t, services.Items, 5)
+	assert.Len(t, services.Items, serviceCount)
```

Comment:
this will break the linux only Helm integration test (https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/.github/workflows/helm-integ-test.yml) since this validator file is common to both the tests, can we add checks for windows only ??

Reply:
Please attach the run of the Linux Helm integration test run as well to ensure these changes are not breaking Linux workflow

Reply:
Attached [test run](https://github.com/aws/amazon-cloudwatch-agent-operator/actions/runs/8332051913) ran for both Linux and Windows. My changes in tests are not breaking Linux tests.